tests/entity_plus.test

```diff
@@ -424,9 +424,10 @@ class EntityPlusMetadataTestCase extends EntityPlusHelperTestCase {
 
     $results = entity_plus_property_query('node', $this->field_name, array('foo', 'bar'));
     $this->assertEqual($results, array($node->nid), 'Queried nodes with a list of possible values.');
-
-    $results = entity_plus_property_query('node', 'author', $GLOBALS['user']);
-    $this->assertEqual($results, array($node->nid), 'Queried nodes with a given auhtor.');
+ 
+    $results = entity_plus_property_query('node', 'author', $GLOBALS['user']->uid);
+    // Backdrop provides 2 nodes with uid == 1 on clean install, meaning that assertEqual will not work
```

Comment:
I'm referring to the sample nodes (one post, one page) that a clean install creates. This means that the $results array contains 3 elements (the two provided nodes, and the node created by the tests, all of which have uid == 1) instead of a single element in Drupal 7.

Reply:
Nice job!